docs/cheat-sheets/java-xxe.mdx

```diff
@@ -0,0 +1,341 @@
+|-------------------------------------------------------------------------------------|----------|-----|------------------|-----|--------|---------|----------|-----|--------|---------|
+|                                                                                     | xml-bomb | dtd | parameter entity | dtd | import | include | document | dtd | import | include |
+| default                                                                             | ✅⚠️       | ❌   | ❌                | ❌   | ✅      | ✅       | ✅        | ❌   | ✅      | ✅       |
+| Default config DTDValidating                                                        | ✅⚠️       | ✅⚠️  | ❌                | ✅⚠️  | ✅⚠️     | ✅⚠️      | ✅⚠️       | ✅⚠️  | ✅⚠️     | ✅⚠️      |
+| Default config XSDValidating                                                        | ✅⚠️       | ✅⚠️  | ❌                | ✅⚠️  | ✅⚠️     | ✅⚠️      | ✅⚠️       | ✅⚠️  | ✅⚠️     | ✅⚠️      |
+| setFeature(XMLConstants.FEATURE_SECURE_PROCESSING, true)                            | ✅⚠️       | ❌   | ❌                | ❌   | ✅      | ✅       | ✅        | ❌   | ✅      | ✅       |
+| setAttribute(XMLConstants.ACCESS_EXTERNAL_DTD, "")                                  | ✅⚠️       | ✅⚠️  | ✅⚠️               | ✅⚠️  | ✅      | ✅       | ✅        | ✅⚠️  | ✅      | ✅       |
+| setAttribute(XMLConstants.ACCESS_EXTERNAL_SCHEMA, "")                               | ✅⚠️       | ❌   | ❌                | ❌   | ✅      | ✅       | ✅        | ❌   | ✅      | ✅       |
+| setAttribute(XMLConstants.ACCESS_EXTERNAL_STYLESHEET, "");                          | n/a      | n/a | n/a              | n/a | n/a    | n/a     | n/a      | n/a | n/a    | n/a     |
+| setFeature("http://apache.org/xml/features/disallow-doctype-decl", true)            | ✅⚠️       | ✅⚠️  | ✅⚠️               | ✅⚠️  | ✅      | ✅       | ✅        | ✅⚠️  | ✅      | ✅       |
+| setFeature("http://xml.org/sax/features/external-general-entities", false)          | ✅⚠️       | ✅   | ❌                | ✅   | ✅      | ✅       | ✅        | ✅   | ✅      | ✅       |
+| setFeature("http://xml.org/sax/features/external-parameter-entities", false)        | ✅⚠️       | ❌   | ✅                | ❌   | ✅      | ✅       | ✅        | ❌   | ✅      | ✅       |
+| setExpandEntities(false)                                                            | ✅⚠️       | ✅   | ❌                | ✅   | ✅      | ✅       | ✅        | ✅   | ✅      | ✅       |
+| setFeature("http://apache.org/xml/features/xinclude", false)                        | ✅⚠️       | ❌   | ❌                | ❌   | ✅      | ✅       | ✅        | ❌   | ✅      | ✅       |
+| setFeature("http://apache.org/xml/features/nonvalidating/load-external-dtd", false) | ✅⚠️       | ❌   | ❌                | ❌   | ✅      | ✅       | ✅        | ❌   | ✅      | ✅       |
```

Comment:
Note to all of these tables (even below) - how do these settings look in code? Maybe we should mark them all as inline code?

For example:
`setFeature(XMLConstants.FEATURE_SECURE_PROCESSING, true)`

Not sure how to mark the: default, Default config DTDValidating, Default config XSDValidating.